test/integration/DCAHubCompanion/multi-pair-swap-with-dex.spec.ts

```diff
@@ -92,10 +91,10 @@ describe('Multi pair swap with DEX', () => {
       let rewardWETH: BigNumber,
         toProvideUSDC: BigNumber,
         toProvideLINK: BigNumber,
-        sentToAgg: BigNumber[],
+        sentWETHToAgg: BigNumber[],
         receivedUSDCFromAgg: BigNumber,
         receivedLINKFromAgg: BigNumber,
-        receivedWETHFromAgg: BigNumber;
+        receivedWETHFromAgg: BigNumber[];
```

Comment:
I realized that 0x is kind of weird, and sometime makes more than one tx, so I fixed it in this PR